packages/server/src/server.ts

```diff
@@ -33,6 +25,9 @@ export class LlamaIndexServer {
     this.workflowFactory = workflow;
     this.componentsDir = options.uiConfig?.componentsDir;
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
TODO: this means we can't run multiple servers per node instance - this is ok, but we should document it.